phpunit.sh

```diff
@@ -34,11 +34,11 @@ if [ ! -d vendor ]; then
 fi
 
 # Start wp-env
-npx -p @wordpress/env wp-env start;
+npx -p @wordpress/env wp-env start
```

Comment:
Feel free to push to this branch if you prefer `;` in bash. I usually don't use them unless there's something else on the same line, like on line 40.